ehitchhiking-ui/src/app/shared/components/registration/registration.component.sass

```diff
@@ -0,0 +1,101 @@
+fieldset
```

Comment:
It is a bad practice to override selector's styles. Could you please add css class to fieldset and override it ?

Reply:
I remove all selector's in styles. Add class to selector's